CHANGELOG.rst

```diff
@@ -2,6 +2,100 @@
 Release Notes
 =============
 
+.. _Release Notes_7.16.0:
+
+7.16.0 / 6.16.0
+======
+
+.. _Release Notes_7.16.0_Prelude:
+
+Prelude
+-------
+
+This release introduces major version 7 of the Datadog Agent, which starts at v7.16.0. The only change from Agent v6 is that
+v7 defaults to Python 3 and only includes support for Python 3. Before upgrading to v7, confirm that any
+custom checks you have are compatible with Python 3. See this ``guide <https://docs.datadoghq.com/agent/guide/python-3/>``_
```

Comment:
I don't understand why there are double backticks here, there are single backticks on the original release note file: https://github.com/DataDog/datadog-agent/blob/master/releasenotes/notes/agent-7-prelude-73db5a64808ff94b.yaml

Anyway, double backticks make this non-valid reST, so let's update this to:
```suggestion
custom checks you have are compatible with Python 3. See this `guide <https://docs.datadoghq.com/agent/guide/python-3/>`_
```